src/store/cache/collections/utils/retrieveCollections.ts

```diff
@@ -90,13 +108,15 @@ export function* retrieveCollections(
       return selected
     },
     retrieveFromSource: function* (ids: ID[]) {
-      // Fetch it
-      const metadatas: UserCollection[] = yield call(
-        AudiusBackend.getPlaylists,
-        userId,
-        ids
-      )
-      // Process deletions
+      let metadatas: UserCollection[]
+
+      if (ids.length === 1) {
+        metadatas = yield call(retrieveCollection, ids[0])
```

Comment:
wanna add a TODO or something to remove this branch when we have our batched endpts?